src/components/Dialog/Dialog.js

```diff
@@ -0,0 +1,32 @@
+import { MDCDialog } from "@material/dialog";
+import dialogTemplate from "./Dialog.html";
+
+export class DialogComponent {
```

Comment:
could you please rename component to LicenseDialogComponent? Because from what I see it it designed to be used for this purpose only

Reply:
Done